### DISCUSSION TITLE
New versions

### DISCUSSION THREAD

**Discussion by Elenalvanova** [dbeaver/dbeaver#20631] [DBeaver Development]
Good afternoon, it became very inconvenient when sorting and filters were removed under the checkbox. There used to be one click, now dancing with tambourines. I had to roll back to the previous version =((

**Comment by LonwoLonwo** [comment]
Hello @Elenalvanova 

Please provide a screen or a video of this situation.

**Comment by Elenalvanova** [comment]
![1](https://github.com/dbeaver/dbeaver/assets/135815890/ccb31d93-fa12-4c06-8fb4-4704f648cbad)

**Reply by LonwoLonwo** [reply]
We have a ticket for that. You can find some hucks there already.

**Comment by Elenalvanova** [comment]
OK, thank you

**Reply by LonwoLonwo** [reply]
https://github.com/dbeaver/dbeaver/issues/19199
